06week/Fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,12 @@
+for (var i=1; i <= 1000; i++)
+{
+if (i % 3 == 0) {
+console.log('Fizz');
+}else if (i % 5 == 0) {
+console.log('Buzz');
+}else if (i % 3 && i % 5 == 0)
```

Comment:
it is too late to check for divisible by both 3 and 5 here since if it is it will be caught in the divisible by 3 at the beginning.